src/cli_shared/cli/mod.rs

```diff
@@ -57,6 +58,9 @@ pub struct CliOpts {
     /// Address used for RPC. By defaults binds on localhost on port 2345.
     #[arg(long)]
     pub rpc_address: Option<SocketAddr>,
+    /// P2P addresses, e.g. `--p2p-address /ip4/0.0.0.0/tcp/12345 --p2p-address /ip4/0.0.0.0/tcp/12346`
```

Comment:
Let's use the word `listening` somewhere. I tend to grep for such keywords, and it's ultimately what it does. Perhaps `--p2p-listening-address`? Otherwise, at first glance, I'd assume it's an address to which Forest should connect.

Reply:
Fixed.